src/cf-typography/src/atoms/date.less

```diff
@@ -0,0 +1,6 @@
+.a-date {
```

Comment:
Should this go in `headings.less`? I don't want to encourage junk like what's happening in the right sidebar here: http://www.consumerfinance.gov/policy-compliance/rulemaking/final-rules/truth-lending-regulation-z-annual-threshold-adjustments-card-act-hoepa/